### PR TITLE
Persistence generate files

### PR DESCRIPTION
**Pull request by Skhu47**

### CODE REVIEW

File: Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt [hunk 1]
```diff
@@ -6,12 +6,21 @@ import java.io.File
 class FileDefinition {
     fun setupCsvFiles(){
         setupTeamCsvFile()
+        setupLocationCsvFile()
```
Comment (MulisaRamukosi) [inline Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt:9]: You need to do this for all tables, and make sure that you add in the relevant field

**Review by KillerPKP — APPROVED** [review]

**Comment by Skhu47** [comment]
Not sure where to add it cause inside of those files I specified the columns

File: Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt [hunk 2]
```diff
@@ -6,12 +6,51 @@ import java.io.File
...
+    private fun setupCommentaryCsvFile(){
+        generateCsvFile(fileName = CsvFileNames.COMMENTARY, columns = "id,matchId,comment,minute\n")
+    }
+
+    private fun setupMatchCsvFile(){
+        generateCsvFile(fileName = CsvFileNames.MATCH, columns = "id,locationId,homeScore,awayScore,date\n")
+    }
+
+    private fun setupLineupCsvFile(){
+        generateCsvFile(fileName = CsvFileNames.LINEUP, columns = "id,matchId,teamId,playerID,type,position,positionNumber\n")
```
Comment (MulisaRamukosi) [inline Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt:35]: write playerID as playerId

File: Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt [hunk 3]
```diff
@@ -6,12 +6,51 @@ import java.io.File
...
+    }
+
+    private fun setupEventCsvFile(){
+        generateCsvFile(fileName = CsvFileNames.EVENT, columns = "id,matchId,teamId,playerID,minute,type\n")
```
Comment (MulisaRamukosi) [inline Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt:39]: similar to above comment

File: Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt [hunk 4]
```diff
@@ -6,12 +6,51 @@ import java.io.File
...
+    }
+
+    private fun setupStatCsvFile(){
+        generateCsvFile(fileName = CsvFileNames.STAT, columns = "id,matchid,home,away,type\n")
```
Comment (MulisaRamukosi) [inline Persistence/src/main/kotlin/storage/fileDefinition/FileDefinition.kt:43]: matchid as matchId

**Review by MulisaRamukosi — COMMENTED** [review]
I left some comments on the files

**Pull request merged**